build/bootstrap_mcs.sh

```diff
@@ -1,179 +1,215 @@
+    if [[ $OS = 'Ubuntu' || $OS = 'Debian' ]]; then
+        sudo apt-get -y update
+        apt-get -y install build-essential automake libboost-all-dev bison cmake \
+        libncurses5-dev libaio-dev libsystemd-dev libpcre2-dev \
+        libperl-dev libssl-dev libxml2-dev libkrb5-dev flex libpam-dev git \
+        libsnappy-dev libcurl4-openssl-dev libgtest-dev libcppunit-dev googletest libsnappy-dev libjemalloc-dev
+    elif [ $OS = 'CentOS Linux' ]; then
+        yum -y install epel-release \
+        && yum -y groupinstall "Development Tools" \
+        && yum -y install bison ncurses-devel readline-devel perl-devel openssl-devel cmake libxml2-devel gperf libaio-devel libevent-devel python-devel ruby-devel tree wget pam-devel snappy-devel libicu \
+        && yum -y install vim wget strace ltrace gdb  rsyslog net-tools openssh-server expect boost perl-DBI libicu boost-devel initscripts jemalloc-devel libcurl-devel gtest-devel cppunit-devel
+    elif [ $OS = 'OpenSuse' ]; then
+        zypper install -y bison ncurses-devel readline-devel libopenssl-devel cmake libxml2-devel gperf libaio-devel libevent-devel python-devel ruby-devel tree wget pam-devel snappy-devel libicu-devel \
+        && zypper install -y libboost_system-devel libboost_filesystem-devel libboost_thread-devel libboost_regex-devel libboost_date_time-devel libboost_chrono-devel \
+        && zypper install -y vim wget strace ltrace gdb  rsyslog net-tools expect perl-DBI libicu boost-devel jemalloc-devel libcurl-devel \
+        && zypper install -y gcc gcc-c++ git automake libtool gtest cppunit-devel
+    fi
+}
+
+stop_service()
+{
+    message "Stopping MariaDB services"
+    systemctl stop mariadb
+    systemctl stop mariadb-columnstore
+}
+
+start_service()
+{
+    message "Starting MariaDB services"
+    systemctl start mariadb-columnstore
+    systemctl start mariadb
+    service mariadb-columnstore status
+    service mariadb status
+}
+
+clean_old_installation()
+{
+    message "Cleaning old installation"
+    rm -rf /var/lib/columnstore/data1/*
+    rm -rf /var/lib/columnstore/data/
+    rm -rf /var/lib/columnstore/local/
+    rm -f /var/lib/columnstore/storagemanager/storagemanager-lock
+    rm -f /var/lib/columnstore/storagemanager/cs-initialized
+    rm -rf /tmp/*
+    rm -rf /var/lib/mysql
+    rm -rf /var/run/mysqld
+    rm -rf $DATA_DIR
+}
+
+build()
+{
+    message "Building sources"
+    local MDB_CMAKE_FLAGS="-DWITH_SYSTEMD=yes
+                     -DPLUGIN_COLUMNSTORE=YES
+                     -DPLUGIN_MROONGA=NO
+                     -DPLUGIN_ROCKSDB=NO
+                     -DPLUGIN_TOKUDB=NO
+                     -DPLUGIN_CONNECT=NO
+                     -DPLUGIN_SPIDER=NO
+                     -DPLUGIN_OQGRAPH=NO
+                     -DPLUGIN_SPHINX=NO
+                     -DBUILD_CONFIG=mysql_release
+                     -DWITH_WSREP=OFF
+                     -DWITH_SSL=system
+                     -DWITH_UNITTESTS=YES
```

Comment:
These two WITH_UNITTESTS, WITH_MICROBENCHMARKS should be commented out by default b/c they call for additional dependencies.

Reply:
And it should be OK! Unittest must be compiled by default, and this extra deps should be also

Reply:
WITH_MICROBENCHMARKS removed